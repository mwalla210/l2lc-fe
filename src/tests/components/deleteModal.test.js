import React from 'react'
import renderer from 'react-test-renderer'
import DeleteModal from '../../components/deleteModal'
jest.mock('../../components/promptModal')

const defaultOptions = {
  title: 'title',
  confirmOnClick: jest.fn(),
  denyOnClick: jest.fn(),
  open: false,
  closeFn: jest.fn(),
  content: 'content'
}

describe('ButtonDefault', () => {
  it ('Renders with snapshot (open: false)', () => {
    let options = Object.assign({}, defaultOptions)
    const component = renderer.create(
      <DeleteModal {...options}/>,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it ('Renders with snapshot (open: true)', () => {
    let options = Object.assign({}, defaultOptions)
    options.open = true
    const component = renderer.create(
      <DeleteModal {...options}/>,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
