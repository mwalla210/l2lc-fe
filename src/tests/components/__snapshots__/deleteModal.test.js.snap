// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDefault Renders with snapshot (open: false) 1`] = `null`;

exports[`ButtonDefault Renders with snapshot (open: true) 1`] = `null`;
